api/Makefile

```
@@ -171,7 +193,7 @@ dev:
 
 .PHONY: local-shell
 local-shell:
-	$(pipenv) shell
+	uv run bash
```

```suggestion
	$(UV) run $(SHELL)
```
This might be better? 
On my mac I have `SHELL=/bin/zsh`
Prolly is set on your linux machine too.